### PR TITLE
6

### PR DESCRIPTION
**Pull request by SamenaBasith**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+const Comments = () => {
+    const [comments, setComments] = useState([])
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        getComments(article_id)
+        .then((comments) => {
+            console.log(comments)
```
Comment (RichardNeat) [inline src/components/Comments.jsx]: please remove stray console logs

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getComments } from '../api';
+
+
+const Comments = () => {
+    const [comments, setComments] = useState([])
+    const { article_id } = useParams();
+
```
Comment (RichardNeat) [inline src/components/Comments.jsx:10]: Can we add loading functionality to the comments please :)

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Awesome work Samena - just a couple of comments below. May also be worth considering what you will load on the page to the user if there happens to be no comments attached to a review. Can they get some kind of feedback?

**Review by lewispricey — APPROVED** [review]
Great work Samena, thanks for adding in the loading!

Good to merge 🚀

**Pull request merged**